account/bitcoin/privkey.go

```diff
@@ -0,0 +1,37 @@
+package bitcoin
+
+import (
+	"github.com/DE-labtory/zulu/account/bitcoin/chaincfg"
+	"github.com/DE-labtory/zulu/keychain"
+	"github.com/DE-labtory/zulu/types"
+	"github.com/btcsuite/btcutil"
+	"github.com/btcsuite/btcutil/hdkeychain"
+)
+
+type PrivKey struct {
+	*hdkeychain.ExtendedKey
+}
+
+func GetPrivKey(key keychain.Key) (*PrivKey, error) {
+	extKey, err := hdkeychain.NewKeyFromString(string(key.PrivateKey))
```

Comment:
#### Problem

The function `func NewKeyFromString(key string) (*ExtendedKey, error)` of the `hdkeychain` package requires a 'serialized private extended key' which is [78 bytes length](https://github.com/btcsuite/btcutil/blob/v1.0.2/hdkeychain/extendedkey.go#L51). But our `keychain` package only provides 32 bytes raw private key.


>The base58-decoded extended key must consist of a serialized payload plus an additional 4 bytes for the checksum.

quote from: https://github.com/btcsuite/btcutil/blob/v1.0.2/hdkeychain/extendedkey.go#L507

#### Suggestion
IMO, use a plain address scheme-e.g. using the function like [`NewAddressPubKeyHash`](https://pkg.go.dev/github.com/btcsuite/btcutil@v1.0.2?tab=doc#NewAddressPubKeyHash)-rather than `hdkeychain` might be easier way to handle key-address derivation.